spock-specs/src/test/resources/snapshots/org/spockframework/smoke/ast/condition/VerifyAllMethodsAstSpec/ignores_conditions_in_overwritten_methods.groovy

```diff
@@ -0,0 +1,28 @@
+public class BaseAssertions extends java.lang.Object {
+
+    @spock.lang.VerifyAll
+    public void isValid(int a) {
+        org.spockframework.runtime.ValueRecorder $spock_valueRecorder = new org.spockframework.runtime.ValueRecorder()
+        org.spockframework.runtime.ErrorCollector $spock_errorCollector = new org.spockframework.runtime.ErrorCollector()
+        try {
+            try {
+                org.spockframework.runtime.SpockRuntime.verifyCondition($spock_errorCollector, $spock_valueRecorder.reset(), 'a >= 0', 5, 9, null, $spock_valueRecorder.record($spock_valueRecorder.startRecordingValue(2), $spock_valueRecorder.record($spock_valueRecorder.startRecordingValue(0), a) >= $spock_valueRecorder.record($spock_valueRecorder.startRecordingValue(1), 0)))
+            }
+            catch (java.lang.Throwable $spock_condition_throwable) {
+                org.spockframework.runtime.SpockRuntime.conditionFailedWithException($spock_errorCollector, $spock_valueRecorder, 'a >= 0', 5, 9, null, $spock_condition_throwable)}
+            finally {
+            }
+        }
+        finally {
+            $spock_errorCollector.validateCollectedErrors()}
+    }
+
+}
+public class SpecificAssertions extends BaseAssertions {
+
+    @java.lang.Override
+    public void isValid(int a) {
+        a > 0
+    }
+
+}
```

Comment:
💬 I am too dumb to figure out how to have the file end with new line :D The transpiled source will not have a newline at the end.

Reply:
Yeah, committing the snapshot files without IntelliJ messing them up accidentally can be tricky.